roles/kubernetes-apps/external_provisioner/local_volume_provisioner/defaults/main.yml

```diff
@@ -7,6 +7,7 @@ local_volume_provisioner_nodelabels: []
 #   - topology.kubernetes.io/zone
 # Levarages Ansibles string to Python datatype casting. Otherwise the dict_key isn't substituted
 # see https://github.com/ansible/ansible/issues/17324
+local_volume_provisioner_use_node_name_only: "false"
```

Comment:
Should be a boolean i.e. `false` not "false"

Reply:
agreed, done